packages/mds-stream/kafka/stream-consumer.ts

```diff
@@ -32,7 +32,13 @@ const createStreamConsumer = async (
   { clientId = 'client', groupId = 'group' }: Partial<KafkaStreamConsumerOptions> = {}
 ) => {
   try {
-    const kafka = new Kafka({ clientId, brokers: getKafkaBrokers() })
+    const brokers = getKafkaBrokers()
+
+    if (!brokers) {
+      return null
+    }
```

Comment:
Do callers of `createStreamConsumer` and `createStreamProducer` now have to check for `null`?

Reply:
`createStreamProducer` and `createStreamConsumer` could always return null (in the case there was an error thrown), so no change there.